docs/manual/docs/cmd/yammer/report/report-groupactivityfilecounts.md

```diff
@@ -0,0 +1,42 @@
+# yammer report groupactivityfilecounts
+
+Gets the number of Yammer messages posted, read, and liked in groups
+
+## Usage
+
+```sh
+yammer report groupactivityfilecounts [options]
```

Comment:
Changed from `groupactivityfilecounts` to `groupsactivitycounts` to match the issue description.